code/repo_sync

```diff
@@ -412,6 +410,18 @@ def curl(url, destinationpath, onlyifnewer=False, etag=None, resume=False):
             return header
         elif http_result == '304':
             return header
+        elif not temp_download_exists and \
+                http_result == '200' and \
+                os.path.isfile(destinationpath) and \
+                (not (targetsize and (targetsize != \
+                 os.path.getsize(destinationpath)))):
```

Comment:
Sure. The one other complex-ish if statement used in that function uses back slashes, so I went with that. The targetsize stuff is meant to do a size comparison, but only if there is a targetsize (in case of a file that, for whatever reason, isn't returned with a content-length header). I'll see what I can do for comments.